__tests__/utils/providers.tsx

```diff
@@ -6,6 +6,8 @@ import { initialRootState, rootReducer, RootState } from '../../libs/store/data/
 import React from 'react'
 import { Auth0Provider } from '@auth0/auth0-react'
 import { IntercomProvider } from 'react-use-intercom'
+import posthog from "posthog-js";
+import { environment } from "apps/console/src/environments/environment";
```

Comment:
You don't have Prettier during the save, please install it, we use single quote in the project

Reply:
I can configure my IDE that way but is not it better that we enforce a prettier at the recommit hook?